example/docker-compose.yml

```diff
@@ -29,27 +31,33 @@ services:
       - .:/conf
       - logs:/home/envoy/logs:rw
     ports:
-      - 8080:80
-  wasm-logs:
+      - 8080:8080
+  envoy-logs:
     depends_on:
       - envoy
+      - wasm-logs
     image: debian:11-slim
     entrypoint: bash
     command:
       - -c
-      - tail -c +0 -f /home/envoy/logs/envoy.log | grep --line-buffered "[critical][wasm]" | tee /home/envoy/logs/ftw.log
+      - tail -c +0 -f /home/envoy/logs/envoy.log
     volumes:
-      - logs:/home/envoy/logs:rw
-  envoy-logs:
+      - logs:/home/envoy/logs:ro
+  wasm-logs:
     depends_on:
       - envoy
-      - wasm-logs
     image: debian:11-slim
     entrypoint: bash
     command:
       - -c
-      - tail -c +0 -f /home/envoy/logs/envoy.log
+      - tail -c +0 -f /home/envoy/logs/envoy.log | grep --line-buffered "[critical][wasm]"
     volumes:
       - logs:/home/envoy/logs:ro
+  tests:
```

Comment:
I think you don't need this container for the example, the only reason this container is here, if I recall correctly is because you can curl the envoy directly from CI? If that is the case I recommend you create another docker-compose file in the e2e folder to include the script. See https://stackoverflow.com/questions/55650342/import-docker-compose-file-in-another-compose-file

On top of that, here I find a circular dependency: e2e depends on example and example depends on e2e (see https://github.com/corazawaf/coraza-proxy-wasm/pull/19/files#diff-fe02bc0074dfa40a0e1b60098921a60d187efae125046c3f008a5cdfa62cf028R60)